package.json

```diff
@@ -54,8 +55,7 @@
     "decimal.js": "^10.2.0",
     "js-yaml": "^3.13.1",
     "json-schema-deref-sync": "^0.13.0",
-    "object.values": "^1.1.1",
-    "swagger-parser": "^6.0.5"
+    "swagger-parser": "^9.0.1"
```

Comment:
This is an important benefit of bumping Node version, up-to-date swagger-parser no longer brings in core-js as a transitive dependency, which should help solve https://github.com/PayU/openapi-validator-middleware/issues/116

Reply:
@kibertoad , if we're going to release a breaking change, i'd like to also handle this together https://github.com/PayU/api-schema-builder/issues/39

Reply:
Sure, I'll work on that, but probably they can be merged separately.

Reply:
@kobik It has been done: https://github.com/PayU/api-schema-builder/pull/47